NuCivic/dkan
# Creation migration after creating harvest source

CIVIC-5133

## Description

Please check the issue https://github.com/NuCivic/dkan/issues/1540

## User story

When an harvest source is created, the migration don't exists.
With this hook integration is created on the fly.

## How to reproduce

1. Create a new node of type harvest source
2. Run drush ms to see the existent migration
